Enforce better naming convention (#45)
infinum/js-linters: CONTRIBUTING.md

# Contributing

## Project setup

> 1. Install root packages `npm install`

Reviewer: ```suggestion
1. Install root packages with `npm install`
```

> 2. Run `npm run bootstrap`

Reviewer: ```suggestion
2. Run `npm run bootstrap` to symlink dependencies
```

## Testing examples

> 1. Navigate to specific example project (angular or nextjs) `cd examples/angular`

Reviewer: ```suggestion
1. Navigate to specific a example project (Angular or Next.js)
```

> 2. Open it in new VSCode instance `code .`

Reviewer: Why does it need to be opened in VSCode?

Author: Should we be more generic and write something like "Open it with you default code editor, e.g. `code .`

Reviewer: I would omit this step completely, since it has nothing to do with the step that comes after.

Author: it does! If you run lint inside monorepo some weird things are happening. You told me that we should test linter rules bu opening them in separate VSCode instance outside monorepo.

Reviewer: I think you are assuming that one needs to open VSCode and use its terminal to run commands. I would not mention VSCode at all. These steps are purely about running commands in any terminal.

The issue we talked about it is  about the ESLint plugin for VSCode that doesn't work in monorepos properly.

> 3. Run lint command `npm run lint`

Reviewer: ```suggestion
3. Run the lint command with `npm run lint`
```

> 4. Optional run test command (only nextjs) `npm run test`

Reviewer: ```suggestion
4. Optionally run the test command (only for Next.js) with `npm run test`
```
